package.json

```diff
@@ -60,15 +60,15 @@
     "@graphql-codegen/typescript-document-nodes": "^2.2.5",
     "@graphql-codegen/typescript-operations": "^2.3.2",
     "@graphql-codegen/typescript-react-apollo": "^3.2.8",
-    "@nrwl/cli": "13.8.3",
-    "@nrwl/cypress": "13.8.3",
-    "@nrwl/eslint-plugin-nx": "13.9.7",
-    "@nrwl/jest": "15.9.2",
-    "@nrwl/linter": "15.8.1",
-    "@nrwl/react": "13.8.3",
-    "@nrwl/tao": "13.8.3",
-    "@nrwl/web": "14.7.5",
-    "@nrwl/workspace": "13.8.4",
+    "@nrwl/cli": "13.8.5",
```

Comment:
This version number, 13.8.5, comes from line 25 above:

    "@nrwl/next": "13.8.5",